### ISSUE TITLE
Simplify config structure

### ISSUE THREAD

**Issue by The-Quill** [The-Quill/SE-Chat-Terminal#33]
I could simplify the config structure down quite a bit.

```
"room_domains": {
    "StackExchange": {
        "8595": "The 2nd Monitor",
        "6697": "Maid Cafe",
        "240": "The Nineteenth Byte"
    },
    "MetaStackExchange": {
        "89": "Tavern on the Meta"
    },
}
```

**Comment by The-Quill** [comment]
Hmm. I want to move toward a more flexible config where you can even just provide the roomid. but then you'd have to do an empty string for the value.

Hmmmmmmm

**Comment by The-Quill** [comment]
Potentially, something like this:

```
        "StackExchange": [
            "8595",
            "6697",
            "240",
            "21",
            "71",
            "35",
            "118",
            "8888",
            "198",
            "36",
            "1888",
            "16",
            "591",
            "20803",
            "14929",
            "16134",
            "22119"
        ],
```

**Comment by The-Quill** [comment]
But then you loose the name connections. I suppose that simplifies the massive amount of messages if debug is on during initial connection.

**Comment by The-Quill** [comment]
I could wrap them in objects, but then the syntax is messy.

**Comment by The-Quill** [comment]
It'd look like that.
[new-config.json.zip](https://github.com/The-Quill/SE-Chat-Terminal/files/166628/new-config.json.zip)

**Comment by The-Quill** [comment]
Fixed in f61d34a0b16d5f030ee787fce91888218f773923
